docs/Installation/Installation_Guide.rst

```diff
@@ -172,6 +172,47 @@ configuring persistent voloumes at 500 Gi/node.
 
    It is a best practice to install Zenko on a fresh cluster.
 
+Pod Disruption Budgets
+----------------------
+
+Zenko relies on several stateful services that require a minimum number
+of pods to function with high availability, with resilence under many
+outage scenarios. Pod disruption budgets set how many pods of a given
+application can safely fail and continue to operate normally before
+Kubernetes disables access to the service.
+
+  .. note::
+
+     If you are installing Zenko in a high-availability production environment,
+     set a pod disruption budget. If you are installing a basic Zenko deployment
+     (for testing or familiarization, for instance) you can skip this step.
+
+The following installed applications allow for configuring disruption
+budgets:
+
+  - MongoDB
+  - Redis
+  - Zenko-Quorum
+
+In a three-node cluster, Zenko configures a ``maxUnavailable`` budget
+of 1 by default. However in larger clusters, this should be increased to
+match the level of high availablity required. For example, a five-node
+cluster can have up to two ``maxUnavailable`` pods per application.
+
+In addition to configuring the node count at install time, you should
+also configure the pod disruption budgets.
+
+.. code:: shell
+
+  --set-string mongodb-replicaset.podDisruptionBudget.maxUnavailable=2
```

Comment:
@wabernatScality if we're adding this to the Installation Guide then I would suggest we explain how to add these to the options.yaml file to be consistent with this doc. The `--set-string` option was just to be consistent with how the README.md explains the options. Both configure the options equally but it simplifies things if we stick to one or the other and not both.

So for example `--set-string mongodb-replicaset.podDisruptionBudget.maxUnavailable=2` would turn into
```yaml
mongodb-replicaset:
  podDisruptionBudget:
    maxUnavailable: 2
```

for the options.yaml format